css/src/components/page.module.css

```diff
@@ -0,0 +1,23 @@
+.Page {
+  background-color: var(--secondary-lightest);
+  width: 100%;
+  height: calc(100dvh - 47px);
```

Comment:
Any specific reason for using this hardcoded value `47px`? Please check if we can remove this with our design tokens